### PR TITLE
For Alexey

### PR DESCRIPTION
**Pull request by olgakil**

### CODE REVIEW

**Comment by cxadmin** [comment]
Scan submitted to Checkmarx

**Comment by cxadmin** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000008&projectid=7)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|5
Medium|624
Low|6872
Informational|24

#### Violation Summary
Severity|Count
---|---
High|4

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|Reflected_XSS_All_Clients|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000008&projectid=7&pathid=819)
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|SQL_Injection|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000008&projectid=7&pathid=821)

File: src/test/java/com/checkmarx/flow/cucumber/integration/scan/ScanSteps.java [hunk 1]
```diff
@@ -200,7 +200,7 @@ public void verifyJsonLoggerAndScanStatus(String repoUrl, String scanStatus) {
         JsonNode node;
         try {
 
-           // AnalyticsReport report = testUtils.getReportNode(ScanReport.OPERATION, ScanReport.class);
+            //AnalyticsReport report = testUtils.getReportNode(ScanReport.OPERATION, ScanReport.class);
```
Comment (alex-ko-dev) [inline src/test/java/com/checkmarx/flow/cucumber/integration/scan/ScanSteps.java]: Please remove the comment.

File: src/test/java/com/checkmarx/flow/cucumber/integration/scan/ScanSteps.java [hunk 2]
```diff
@@ -200,7 +200,7 @@ public void verifyJsonLoggerAndScanStatus(String repoUrl, String scanStatus) {
...
             node = testUtils.getReportNode(ScanReport.OPERATION);
```
Comment (alex-ko-dev) [inline src/test/java/com/checkmarx/flow/cucumber/integration/scan/ScanSteps.java]: Did you try deserializing to ScanReport?

**Comment by cxadmin** [comment]
Scan submitted to Checkmarx

**Comment by cxadmin** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000009&projectid=7)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|5
Medium|624
Low|6827
Informational|24

#### Violation Summary
Severity|Count
---|---
High|4

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|Reflected_XSS_All_Clients|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000009&projectid=7&pathid=165)
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|SQL_Injection|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000009&projectid=7&pathid=167)

**Comment by cxadmin** [comment]
Scan submitted to Checkmarx

**Comment by cxadmin** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000011&projectid=7)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|5
Medium|627
Low|6828
Informational|24

#### Violation Summary
Severity|Count
---|---
High|4

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|Reflected_XSS_All_Clients|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000011&projectid=7&pathid=171)
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|SQL_Injection|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000011&projectid=7&pathid=173)

**Comment by cxadmin** [comment]
Scan submitted to Checkmarx

**Comment by cxadmin** [comment]
Scan submitted to Checkmarx

**Comment by cxadmin** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000012&projectid=7)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|5
Medium|627
Low|6828
Informational|24

#### Violation Summary
Severity|Count
---|---
High|4

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|Reflected_XSS_All_Clients|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000012&projectid=7&pathid=171)
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|SQL_Injection|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000012&projectid=7&pathid=173)

**Comment by cxadmin** [comment]
### Checkmarx scan completed
[Full Scan Details](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000013&projectid=7)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|5
Medium|627
Low|6828
Informational|24

#### Violation Summary
Severity|Count
---|---
High|4

#### Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|Reflected_XSS_All_Clients|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000013&projectid=7&pathid=171)
[88](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L88) [89](https://github.com/checkmarx-ltd/cx-flow/blob/Orly_fixes_pull_request/src/test/resources/cucumber/data/input-code-samples/DOS_Login.java#L89) |High|SQL_Injection|src/test/resources/cucumber/data/input-code-samples/DOS_Login.java|[Checkmarx](http://EC2AMAZ-JP2I9QF/CxWebClient/ViewerMain.aspx?scanid=1000013&projectid=7&pathid=173)

**Pull request merged**